<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="main_tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_recent">
       <attribute name="title">
        <string>RECENT</string>
       </attribute>
       <layout class="QVBoxLayout" name="recent_layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="clock_label">
            <property name="styleSheet">
             <string notr="true">color: rgb(102, 102, 102);
font: 18pt &quot;Calibri&quot;;</string>
            </property>
            <property name="text">
             <string>CLOCK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_taf">
       <attribute name="title">
        <string>TAF</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="taf_table">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>报文内容</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>发布时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>查询</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>其他</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_metar">
       <attribute name="title">
        <string>METAR</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="metar_table">
          <column>
           <property name="text">
            <string>类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>报文内容</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="post_menu">
    <property name="title">
     <string>编发</string>
    </property>
    <addaction name="taf_action"/>
    <addaction name="sigmet_action"/>
    <addaction name="trend_action"/>
   </widget>
   <widget class="QMenu" name="setting_menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="warn_action"/>
    <addaction name="setting_action"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="about_action"/>
   </widget>
   <addaction name="post_menu"/>
   <addaction name="setting_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="taf_action">
   <property name="text">
    <string>预报</string>
   </property>
  </action>
  <action name="trend_action">
   <property name="text">
    <string>趋势</string>
   </property>
  </action>
  <action name="sigmet_action">
   <property name="text">
    <string>重要气象情报</string>
   </property>
  </action>
  <action name="setting_action">
   <property name="text">
    <string>首选项</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="warn_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>告警</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
